neuromation/cli/job.py

```diff
@@ -168,8 +183,8 @@ def job() -> None:
     multiple=True,
     help="Mounts directory from vault into container. "
     "Use multiple options to mount more than one volume. "
-    "--volume=HOME is an alias for storage://~:/var/storage/home:rw and "
-    "storage://neuromation/public:/var/storage/neuromation:ro",
+    f"--volume=HOME is an alias for storage://~:{STORAGE_MOUNTPOINT}/home:rw and "
```

Comment:
String fomatting operations in docstrings decrease the code readability, not increase it.

Reply:
fixed